docs/docs/contributing/contributing.md

```diff
@@ -0,0 +1,99 @@
+---
+layout: default
+title: Contributing
+nav_order: 4
+has_children: true
+permalink: /contributing
+---
+
+# Contributing
+{: .no_toc }
+
+## Table of contents
+{: .no_toc .text-delta }
+
+1. TOC
+{:toc}
+
+---
+
+# Open Container Initiative Contributions
+
+Before you are ready to contribute, make sure that you understand:
+
+ 1. What are the different OCI specifications, on an abstract level?
+ 2. What are applied usage examples of the specifications?
+ 3. How do I ask for help or clarification? 
+
+You can have a gentle introduction to containers [here]({{ site.baseurl }}introduction/container-images/), and 
+read about the OCI types [here]({{ site.baseurl }}introduction/oci-types/). 
+
+## Kinds of contribution?
+
+When you understand what a container is, and what the goals are behind each of the specifications 
+provided by OCI, you are then in the right mindset to contribute. Generally, you might fit into one of the following scenarios:
+
+ 1. **Definition** You have a goal or need that is not addressed by a specification.
+ 2. **Documentation** An idea or concept is not explained enough for you to use it.
+ 3. **Feature Request** You have a new idea for an extension to a specification.
+
+To serve these various contribution use cases, we have an [issue tracker](https://www.github.com/vsoch/oci-contribution/issues) with templates to guide you through the contribution process.
+Each contribution request, whether it be asking a question or a feature, will happen via
+discussion on the issue board. The final decision for a request might result in the opening of new
+issues on other [opencontainers](https://www.github.com/opencontainers) boards, per the
+collaborative discussion and decision of those involved. 
+
+We have provided several templates to help guide you in your question, issue, or feature request. [Open an issue](https://www.github.com/vsoch/oci-contribution/issues) to get started with the process.
+Now let's walk through what interactions and discussions on the issue tracker might look like.
+
+## How do I Contribute?
+
+### 1. Develop an Idea
+
+You are on this page in first place, and reading about how to contribute, because you found a hole that isn't addressed by the specification, or have an idea for an extension to make it better. The first thing you should do is grab your computer, go to a comfortable working spot, and write. What is your use case or need? How did you try to use current software that implements OCI (and how did it fail?) Tell your problem like a story. It will be easier for experts and non-experts alike to follow your train of thought, and understand the issue you are trying to address. If you fail to capture their understanding off the bat you will have a very hard time getting support for change.
```

Comment:
comfy spot :-)

Reply:
/s/will/may/
/s/very//

The group can be friendly sometimes :-)

Reply:
I have the very disease. Very bad.